javascript/javascript2/week3/TheFastestPresserInThisRealm/index.html

```diff
@@ -0,0 +1,38 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+    <h1>Fastest presser</h1>
+    <div class="wrapper">
+        <!-- <input type="numder" placeholder="set seconds" onclick="console.log(this.value)"> -->
+        <input type="numder" placeholder="Set seconds">
```

Comment:
typo here .  numder should be number and will prevent the user to type text

Reply:
and you can add min and max attributes so it will be a limited input type number  fx : <input type="number" placeholder="Set seconds" min="1" max="60">

Reply:
Thanks for review and for tips, they made a code clearer and more readability.